x/custom/model.go

```diff
@@ -0,0 +1,103 @@
+package custom
+
+import (
+	"github.com/iov-one/tutorial/morm"
```

Comment:
I am not sure about this import. I understand that the purpose of this repository is to provide a bootstrap repository for a project. A starter code for an application that will be deployed to production one day. If this is the case, I think it does not look good if we import `tutorial`. 
If `morm` is a good functionality that is missing in `weave`, maybe we should add it there?

Reply:
I think `morm` is intuitive to use if you're not used to KV databases. Would be great if we add it to `weave`. This is the related open issue https://github.com/iov-one/weave/issues/824.

Reply:
I have creted https://github.com/iov-one/weave/issues/924